src/api/json/catalog.json

```diff
@@ -7308,6 +7308,12 @@
       "fileMatch": ["**/*winutil*/config/preset.json"],
       "url": "https://json.schemastore.org/winutil-preset.json"
     },
+    {
+      "name": "GitHub Actions typing",
+      "description": "Describe types of inputs and outputs for GitHub Actions' actions.",
+      "fileMatch": ["**/action-types.yml", "**/action-types.yaml"],
```

Comment:
To be checked: if `**` in the pattern also matches "no directory", that is: if the schema will work also for files in the root dir of some repo.

Reply:
Checked that in IntelliJ, `**/` is not needed to match files in some directory.

Reply:
@madskristensen @hyperupcall please advise. I want the schema to apply for every file named `action-types.yml` or `action-types.yaml`, regardless of the location.

Reply:
When I prepared a PR for this some time ago, I indeed hat just
```plain
"fileMatch": ["action-types.yml", "action-types.yaml"],
```
like many other entries do